tests/functional/object/mcg/ui/test_mcg_ui.py

```diff
@@ -414,3 +415,27 @@ def test_obc_creation_and_deletion(
         obc_ui_obj.delete_obc_ui(obc_name, delete_via)
 
         assert test_obc.check_resource_existence(should_exist=False)
+
+
+@black_squad
+class TestBucketCreate:
+    def test_bucket_create(self, setup_ui_class_factory):
+        """
+        Test bucket creation functionality in UI.
+
+        Creates both OBC and S3 buckets, then creates a folder in one of them.
+        Verifies basic bucket and folder creation workflows through the UI.
+
+        """
+        setup_ui_class_factory()
+        bucket_ui = BucketsTab()
+        bucket_ui.nav_object_storage_page()
+        bucket_ui.create_bucket_ui("obc")
+        time.sleep(15)
+        bucket_ui.nav_object_storage_page()
+        bucket_ui.create_bucket_ui("s3")
+        time.sleep(15)
+        bucket_ui.nav_object_storage_page()
+        bucket_ui.create_folder_in_bucket()
+        bucket_ui.nav_buckets_page()
+        bucket_ui.get_buckets_list()
```

Comment:
is it some kind of a libtest? do we want to add assertions to validate bucket list?

Reply:
its WIP. going to compare list by GUI to list by CLI